marketplace/netlify-build/extension.json

```diff
@@ -0,0 +1,38 @@
+{
+  "id": "netlify-button",
+  "name": "Netlify Build and Preview",
+  "src": "http://localhost:1234",
+  "sidebar": true,
+  "parameters": {
+    "installation": [
+      {
+        "id": "netlifySiteIds",
+        "type": "Symbol",
+        "name": "Comma seperated Netlify Site Id list (max 3)",
+        "description": "Example: 14234e3,143248,35233",
+        "required": true
+      },
+      {
+        "id": "buildHookUrls",
+        "type": "Symbol",
+        "name": "Comma seperated Netlify build hook URLs (max 3)",
+        "description": "Example: https://api.netlify.com/build_hooks/xyz,https://api.netlify.com/build_hooks/abc",
+        "required": true
+      },
+      {
+        "id": "names",
+        "type": "Symbol",
+        "name": "Comma seperated Website name list (max 3)",
+        "description": "Example: My Site,My Other Site",
+        "required": true
+      },
+      {
+        "id": "netlifySiteUrls",
+        "type": "Symbol",
+        "name": "Comma seperated Netlify site URLs (max 3)",
+        "description": "Example: https://firefly-12345.netlify.com",
```

Comment:
In all examples provide the same number of values, I think 2 is fine

Reply:
Done.